src/app/services/work-item.service.ts

```diff
@@ -247,7 +247,7 @@ export class WorkItemService {
    */
   getWorkItemById(id: string): Observable<WorkItem> {
     if (this._currentSpace) {
-      return this.http.get(this._currentSpace.links.self + '/workitems/' + id)
+      return this.http.get(this._currentSpace.links.self.split('/spaces/')[0] + '/workitems/' + id)
```

Comment:
In case user directly launch to the details page, we load the detail page and the list page in parallel, at that point you might not have any previous reference of the work item. 
Also, this is a way to keep the details page independent of any other page.